src/lib.rs

```diff
@@ -240,18 +239,23 @@ async fn handle_server_fns(req: IncomingRequest, resp_out: ResponseOutparam) {
     let url = url::Url::parse(&url(&req)).unwrap();
     let mut path_segs = url.path_segments().unwrap().collect::<Vec<_>>();
 
-    let payload = loop {
+    let (payload, res_parts, runtime) = loop {
         if path_segs.is_empty() {
             panic!("NO LEPTOS FN!  Ran out of path segs looking for a match");
         }
 
         let candidate = path_segs.join("/");
-
         if let Some(lepfn) = leptos::leptos_server::server_fn_by_path(&candidate) {
             // TODO: better checking here - again using the captures might help
             if pq.starts_with(lepfn.prefix()) {
+                // Need to create a Runtime and provide some expected values
+                let runtime = create_runtime();
+                //provide_context(req.clone());
+                let res_parts = ResponseOptions::default();
```

Comment:
The default `ResponseOptions` currently sets a `Content-Type: text/html` header, which won't be right for server function responses, although I don't know if clients will care!

We can change the default (and update the view code to set the HTML type), or provide another constructor that doesn't set the header, or do nothing since it works regardless.

Reply:
Oh! That's different from what I'm used to. We've been adding that header manually in each render function, and defaulting to blank

Reply:
Sure, no worries - thanks for the guidance!